### ISSUE TITLE
Populating the url parameter with "<iframe" throws a TypeError

### ISSUE THREAD

**Issue by TerenceJefferies** [radiovisual/get-video-id#102]
**Describe the bug**  
When using the string "<iframe" as the `url` parameter in the `getVideoId` method, a TypeError is thrown. I believe the package may be removing tags, which it then tries to trim.

**To Reproduce**
Detailed steps to reproduce the behaviour:
1. Call `getVideoId` with the url string: `<iframe`

**Expected behaviour**  
A non-valid id is returned, eg.`{ id: null, service: null }` as other invalid parameters do.

**Environment**  
Tested in:

1. Node v14.19.0
2. Browser (Chrome, v104)

**Module version**  
v3.6.4

**Actual Error:**

```shell
TypeError: Cannot read property 'trim' of undefined
```

**Comment by radiovisual** [comment]
Hello, @TerenceJefferies, thanks for reporting! Can you show me the exact code snippet which throws this error for you? Because when I run this module, it works, and the tests which have the use case you described (where `<iframe` is the beginning of the string) are all passing (you can see an example of a test [here](https://github.com/radiovisual/get-video-id/blob/master/__tests__/youtube.test.js#L60-L68). Perhaps the code snippet could offer a clue.

**Comment by TerenceJefferies** [comment]
@radiovisual Hi, of course, I am using this snippet:

```javascript
const getVideoId = require("get-video-id")

console.log(getVideoId("<iframe"))
```

**Comment by TerenceJefferies** [comment]
As a use-case scenario: I have an input field that accepts an embed code, should the user start typing manually, an exception is thrown once they get to this part of the string.

**Comment by radiovisual** [comment]
@TerenceJefferies problem should be fixed in version 3.6.5. Happy coding!
